v4l2loopback.c

```diff
@@ -2696,16 +2596,8 @@ static const struct v4l2_file_operations v4l2_loopback_fops = {
 static const struct v4l2_ioctl_ops v4l2_loopback_ioctl_ops = {
 	// clang-format off
 	.vidioc_querycap		= &vidioc_querycap,
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 29)
 	.vidioc_enum_framesizes		= &vidioc_enum_framesizes,
 	.vidioc_enum_frameintervals	= &vidioc_enum_frameintervals,
-#endif
-
-#ifndef HAVE__V4L2_CTRLS
-	.vidioc_queryctrl		= &vidioc_queryctrl,
-	.vidioc_g_ctrl			= &vidioc_g_ctrl,
-	.vidioc_s_ctrl			= &vidioc_s_ctrl,
```

Comment:
You should not remove these op func, otherwise clients will not be able to enumerate supported V4L2 controls. They're not ever deprecated in kernel.

Reply:
Those were only defined if HAVE__V4L2_CTRLS wasn't defined. Since I took out the ifndef and effectively acted as is that was always defined, I assumed those were unneeded. Was there a bug in the original code in that it should have been an ifdef instead of ifndef?